src/CreateDemonPage.js

```diff
@@ -14,6 +15,14 @@ export default function CreateDemonPage() {
   async function handleSubmit(e) {
     e.preventDefault();
 
+    const monster = await createDemon({// eslint-disable-line
+      demonName: name,
```

Comment:
the properties on the demon object here (demonName, demonType, etc.) don't directly match the properties being called in your DemonListPage.js - which might be why the list page isn't loading up the details for the demons added